metrotranscom/doorway
# fix: remove ability to edit close and lottery dates on a published listing

# Pull Request Template

## Issue Overview

This PR addresses #570


## Description

Disables ability to edit due dates for active listings by non admin users. Also blocks switching between FCFS and Lottery

## How Can This Be Tested/Reviewed?

1. Non-admin should still be able to create listing with dates, and application type.
2. Non-admin should still be able to edit listing (when dates not changed)
3. Backend technically should not throw an error, as on frontend fields are blocked (only way to check it in app is to switch `disableDueDates` and `disableDueDate` to `false`).

## Checklist:


## Reviewer Notes:

Steps to review a PR:

- Read and understand the issue, and ensure the author has added QA notes
- Review the code itself from a style point of view
- Pull the changes down locally and test that the acceptance criteria is met
- Also review the acceptance criteria on the Netlify deploy preview (noting that these do not yet include any backend changes made in the PR)
- Either explicitly ask a clarifying question, request changes, or approve the PR if there are small remaining changes but the PR is otherwise good to go

## On Merge:

If you have one commit and message, squash. If you need each message to be applied, rebase and merge.
